src/node_options.cc

```diff
@@ -761,6 +761,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "exclude files from coverage report that match this glob pattern",
             &EnvironmentOptions::coverage_exclude_pattern,
             kAllowedInEnvvar);
+  AddOption("--test-global-setup", 
+            "run this script before running tests",
```

Comment:
This doesn't feel accurate IMHO, as it could run a script after running the tests as well

Reply:
Also, this should be added to `node.1` as well
https://github.com/nodejs/node/blob/fab6906c5d1c16b046187485a7cb136b3659d42b/doc/node.1

Reply:
> This doesn't feel accurate IMHO, as it could run a script after running the tests as well 
I totally agree, this needs a better description.
What do you think about the flag name? should we coming up with a better naming ? 

Should we follow mocha's "global-fixtures" ? https://mochajs.org/#global-fixtures ?

Reply:
In `jest` and `vitest` it is called `globalSetup`, so IMHO that is a good name, but I don't feel strongly about it

Reply:
I've seen `jest globalTeardown`, IMHO, having a single file could improve devEx!

Reply:
> IMHO, having a single file could improve devEx!

I agree 🙂 
That is also how vitest works